doc/source/configuration/cephadm.rst

```diff
@@ -308,6 +308,128 @@ should be used in the Kolla Manila configuration e.g.:
 
   manila_cephfs_filesystem_name: manila-cephfs
 
+RADOS Gateways
+--------------
+
+RGWs are defined with the following:
+
+.. code:: yaml
+
+  cephadm_radosgw_services:
+ - id: myrgw
+   count_per_host: 1
+   spec:
+     rgw_frontend_port: 80
+
+Ceph RGWs require additional configuration for the following features:
+
+  * Support both S3 and Swift APIs.
+
+  * Authenticate user access via Keystone.
+
+  * Allow cross-project and public object access.
+
+The set of commands below configure all of these.
+
+.. code:: yaml
+
+  # Append the following to cephadm_commands_post:
+  - "config set client.rgw rgw_content_length_compat true"
```

Comment:
Is there not a better way to set this config e.g. in a file somewhere?

Reply:
Pretty sure that all cephadm configuration is done via commands like this. For example, the osd registration in the cephadm ansible collection: https://github.com/stackhpc/ansible-collection-cephadm/blob/380ad04ea7257c193784c61ce5c9b117ef3bee94/roles/cephadm/tasks/osds.yml#L12